spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('Intervention component', function () {
+  var FIXTURE
+
+  function loadInterventionComponent () {
```

Comment:
Having this as a separate function seems unnecessary - it's two lines that could be put into the `beforeEach` function quite easily.